docs/raw/push-notification-server.md

```diff
@@ -0,0 +1,766 @@
+and save some bandwidth in cases where is not filtering by public keys 
+
+### What's the  bandwidth impact for this?
+
+Generally speaking, for each 1-to-1 message and group chat message you will sending
+1 and `number of participants` push notifications. This can be optimized if 
+multiple users are using the same push notification server. Queries have also 
+a bandwidth impact but they are made only when actually needed
+
+### What's the information disclosed?
+
+The data disclosed with each message sent by the client is above, but for a summary:
+
+When you register with a push notification service you may disclose:
+
+1) Your public key
+2) Which devices you have
+3) The hash of the chat_ids you want to filter out
+4) The has of the public keys you are interested/not interested in
+
+When you query a notification service you may disclose:
+
+1) Your public key
+2) The fact that you are interested in sending push notification to a given user
+
+Effectively this is fairly revealing if the user has a whitelist implemented.
+Therefore sending notification should be optional.
+
+### What prevents a user from generating a random key and getting an access token and spamming?
+
+Nothing really, that's the same as the status app as a whole. the only mechanism that prevents
+this is using a white-list as described above, but that implies disclosing your true identity to the push notification server.
+
+### Why not 0-knowledge proofs/quantum computing
+
+We start simple, we can iterate
+
+### How to handle backward/forward compatibility
+
+Most of the request have a target, so protocol negotiation can happen. We cannot negotiated
+the advertisement as that's effectively a broadcast, but those info should not change and we can
+always accrete the message.
+
+### Why ack_required?
+
+That's necessary to avoid duplicated push notifications.
+
+Deduplication of the push notification is done on the client side, to relive a bit 
+of centralization and also in order not to have to modify gorush. 
+
+### Can I run my own node?
+
+Sure, the methods allow that
+
+### Can I register with multiple nodes for redundancy
+
+Yep
+
+### What does my node disclose?
+
+Your node will disclose the IP address is running from, as it makes an HTTP post to 
+gorush. A waku adapter could be used, but please not now.
+
+### Does this have high-reliability requirements?
+
+The gorush server yes, no way around it.
+
+The rest, kind of, at least one node having your token needs to be up for you to receive notifications.
+But you can register with multiple servers (desktop, status, etc) if that's a concern.
+
+### Can someone else (i.e not status) run this?
+
+Push notification servers can be run by anyone. Gorush can be run by anyone I take,
+but we are in charge of the certificate, so they would not be able to notify status-clients.
+
+## Changelog
+
+### Version 0.1
+
+Released [May 22, 2020](https://github.com/status-im/specs/commit/664dd1c9df6ad409e4c007fefc8c8945b8d324e8)
```

Comment:
Is this true? I thought that this spec was only drafted this month. Maybe just a copypasta from another file?

Reply:
No, that is indeed a copypasta, I will fill it in once ready to merge :)